modules/csharp/partials/release-notes/couchbase-mobile-csharp-release-note.3.1.0.adoc

```diff
@@ -160,7 +160,60 @@ include::ROOT:cbl-whatsnew.adoc[tag=scopes-and-collections]
 
 === Known Issues
 
-None for this release
+The following Third Party limitations apply for this version.
+
+==== Using `TLSIdentity` with .NET 6 iOS and Mac Catalyst
+
+In order to be able to use the `TLSIdentity` class, you must:
```

Comment:
Let's add a small correction here.  You can still use the `TLSIdentity.GetIdentity(X509Certificate2Collection)` method, but that is the only one.  The limitation here is for asking Couchbase Lite to perform cert / key storage on the user's behalf, whereas this method is for using a cert and key that they manage themselves.  

> In order to be able to fully use the `TLSIdentity` class, you must perform a series of steps.  Without these steps the only API available will be `TLSIdentity.GetIdentity(X509Certificate2Collection)`.  To use the rest of the API, you must: